runtime/interop/premine-state.go

```diff
@@ -206,7 +205,8 @@ func (s *PremineGenesisConfig) processDeposits(ctx context.Context, g state.Beac
 	if _, err = helpers.UpdateGenesisEth1Data(g, deposits, g.Eth1Data()); err != nil {
 		return err
 	}
-	_, err = b.ProcessPreGenesisDeposits(ctx, g, deposits)
+	// TODO: should be updated when electra E2E is updated
```

Comment:
I didn't update this to use the new process electra deposits because electra will process deposits differently, I guess I need a new way to add them here...